recover_handler.go

```diff
@@ -15,11 +15,6 @@ func Recover(handler http.Handler) http.Handler {
 			defer func() {
 				if rec := recover(); rec != nil {
 					msg, stack := extractPanic(rec)
-					log.WithFields(log.Fields{
```

Comment:
I think this one is useful to keey, because it actually contains the stacktrace. If the error was caught earlier it would not end up here, so this is a logging-of-last-resort.